devops/binary-footprint-baseline/baseline.json

```diff
@@ -0,0 +1,56 @@
+[
+  {
+    "distro": "ubuntu-18.04",
+    "arch": "x86_64",
+    "sizeBytes": 1772612,
```

Comment:
Does this mean we will need to update our baselines over time as osconfig grows so that CI will pass?

Reply:
yes, thats correct. We will assess to update the baselines then, its a sort of gate we will use to ensure no builds are produced that have too much of a size delta to be flagged. Its at that point we can update the baselines in order to get the gates passing again